lms/templates/dashboard.html

```diff
@@ -33,6 +33,10 @@
   <%static:include path="dashboard/${template_name}.underscore" />
 </script>
 % endfor
+% if course_entitlements:
+    <script type="text/javascript" src="${static.url('common/js/vendor/popper.js')}"></script>
+    <script type="text/javascript" src="${static.url('common/js/vendor/bootstrap.js')}"></script>
+% endif
```

Comment:
Please add a comment noting that this is included here as a temporary fix to an issue with Bootstrap, linking to the ticket to update the way this is implemented.